app/javascript/components/App.js

```diff
@@ -28,9 +29,24 @@ class App extends Component {
   };
 
   createAuction = (auction) => {
-    fetch("/auction_items", { method: "POST", body: auction });
+    fetch("/auction_items", { method: "POST",headers:{'Content-Type': 'application/json'}, body: JSON.stringify({auction_item:auction}) });
```

Comment:
This needs some indentation and line spacing love.

Reply:
@hawk198723 If you install this extension and enable format on save in the VSCode settings, it should take care of the spacing and indentation automatically:  https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
thank you, Zan, I already had it, I think I get some code from ATOM and cause the issue, I already fix the indentation.